docs/tutorials/proposals/referral-program-proposal.md

```diff
@@ -0,0 +1,226 @@
+* Familiarity with [governance on Vega](../../concepts/governance.md)
+
+## Anatomy of a referral program proposal
+The fields below all need to be defined to enable the referral program or replace an existing one. 
+
+If you are suggesting a replacement referral program, you'll need to include all the fields, even if you don't want to change their values. Just use the existing values from the current referral program.
+
+**End of program timestamp**: Date and time after which, when the current epoch ends, the program will end and benefits will be disabled.
+
+**Window length**: Number of epochs over which to evaluate a referral set's running notional taker volume.
+
+#### Benefit tier fields
+
+| Benefit tier field | Description | Accepted values |
+| ----------- | ----------- | ----------- |
+| `benefitTiers` | List of values defining the reward and discount factors for the program | Holds the details of each benefit tier, listed below. Maximum of <NetworkParameter frontMatter={frontMatter} param="referralProgram.maxReferralTiers" hideName={true}/> tiers |
+| `minimumRunningNotionalTakerVolume` | The notional volume of aggressive trades that a trader is required to have across the aggregation window, to access this tier | Whole number, decimals allowed, greater than 1 |
+| `minimumEpochs` | Required number of epochs that a referee must have been in a referral set to access the benefits in this tier | Integer greater than 0 | Integer greater than 0 |
+| `referralRewardFactor` | Proportion of the referee's taker fees that will be rewarded to the referrer | Whole number, decimals allowed, greater than or equal to 0, and less / equal to <NetworkParameter frontMatter={frontMatter} param="referralProgram.maxReferralRewardFactor" hideName={true}/> |
+| `referralDiscountFactor` | Proportion of each referee's taker fees to be discounted, will be converted to a percentage | Must be greater than or equal to 0 and less than / equal to <NetworkParameter frontMatter={frontMatter} param="referralProgram.maxReferralDiscountFactor" hideName={true}/> |
+
+#### Staking tier fields
+
+| Staking tier field | Description | Accepted values |
+| ----------- | ----------- | ----------- |
+| `stakingTiers` | List of values defining the multipliers to be used for referrals | Holds the details for each benefit tier, listed below. Maximum of <NetworkParameter frontMatter={frontMatter} param="referralProgram.maxReferralTiers" hideName={true}/>|
+| `minimumStakedTokens` | Required number of VEGA tokens a referrer must have associated to their Vega key to receive the reward multiplier | Integer greater than or equal to 0 |
```

Comment:
Have double and triple checked, this is an integer greater than or equal to `1`.